src/module.ts

```diff
@@ -116,15 +117,26 @@ export default defineNuxtModule<ModuleOptions>({
     // Create an Ionic config file if it doesn't exist yet
     const ionicConfigPath = join(nuxt.options.rootDir, 'ionic.config.json')
     if (!existsSync(ionicConfigPath)) {
+      // Look for any `capacitor.config.{json,js,ts}` and load it
+      const { config } = await loadConfig<CapacitorConfig>({
+        cwd: nuxt.options.rootDir,
+        name: 'capacitor',
+        rcFile: false,
+        jitiOptions: {
+          interopDefault: true,
+          esmResolve: true,
+        },
+      })
+
       await fsp.writeFile(
         ionicConfigPath,
         JSON.stringify(
           {
-            name: await readPackageJSON(nuxt.options.rootDir).then(
-              ({ name }) => name || 'nuxt-ionic-project'
-            ),
-            integrations: {},
-            type: 'vue',
+            name: config?.appName || 'nuxt-ionic-project',
+            integrations: {
+              capacitor: {},
+            },
+            type: 'vue-vite',
```

Comment:
are these changes to the written file definitely correct, bearing in mind the user might not use capacitor? And I'm not sure `vue-vite` is right:

> To support this, we’ve introduced a new project type in the ionic.config.json, vue-vite and react-vite. These new project types will tell the CLI that it should call Vite when you run ionic serve or ionic build. 
> https://ionic.io/blog/ionic-cli-v7